air/src/options.rs

```diff
@@ -168,17 +170,17 @@ impl From<ProvingOptions> for WinterProofOptions {
 /// - `expected_cycles` specifies the number of cycles a program is expected to execute.
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub struct ExecutionOptions {
-    max_cycles: u32,
-    expected_cycles: u32,
+    max_cycles: RowIndex,
+    expected_cycles: RowIndex,
```

Comment:
Agreed. While an inclusive maximum is effectively the same as the index it relates to, it reads strangely because an index is not a count.

If renaming the variables to max_row (or similar) doesn't make sense, then would suggest to use u32 instead.

Reply:
Let's change these back to `u32` values.